Src/Base/AMReX_TinyProfiler.cpp

```diff
@@ -713,6 +779,9 @@ TinyProfiler::PrintStats (std::map<std::string,Stats>& regstats, double dt_max)
 #endif
         }
         amrex::OutStream() << hline << "\n\n";
+        if (print_other_procstat) {
+            allprocstats.pop_back();
```

Comment:
`allprocstats` is a local variable in this function. So is there any reason to call `pop_back` here?

Reply:
Just to be symmetric with the call above, I will remove it.